central/networkbaseline/manager/manager_impl.go

```diff
@@ -247,24 +256,25 @@ func (m *manager) processFlowUpdate(flows map[networkgraph.NetworkConnIndicator]
 			continue
 		}
 		if conn.SrcEntity.Type == storage.NetworkEntityInfo_DEPLOYMENT {
-			peerName := m.lookUpPeerName(conn.DstEntity)
-			if peerName != "" {
+			info := m.lookUpPeerInfo(conn.DstEntity)
+			if info.name != "" {
 				m.maybeAddPeer(conn.SrcEntity.ID, &networkbaseline.Peer{
 					IsIngress: false,
 					Entity:    conn.DstEntity,
-					Name:      peerName,
+					Name:      info.name,
 					DstPort:   conn.DstPort,
 					Protocol:  conn.Protocol,
+					CidrBlock: info.cidrBlock,
 				}, modifiedDeploymentIDs)
 			}
 		}
 		if conn.DstEntity.Type == storage.NetworkEntityInfo_DEPLOYMENT {
-			peerName := m.lookUpPeerName(conn.SrcEntity)
-			if peerName != "" {
+			peerInfo := m.lookUpPeerInfo(conn.SrcEntity)
```

Comment:
Nit: Is there a reason you are switching between `info` and `peerInfo` as the result var for `lookUpPeerInfo`? 
Here, it is `peerInfo`, but in [L259](https://github.com/stackrox/stackrox/pull/5194/files#diff-460fa46f51a7facf2085f418a1f9b6bbec93f5046f1efb205d68bc702ac871acR259) and [L443](https://github.com/stackrox/stackrox/pull/5194/files#diff-460fa46f51a7facf2085f418a1f9b6bbec93f5046f1efb205d68bc702ac871acR443) it is `info`. 

And one more thought: How about calling it just `peer`, as it is info we're saving about a peer? That would make it `peer.name` and `peer.cidr` or `peer.cidrBlock`, which I (personally) find very pleasing 😄 
I am aware that `peer` as a var name is already taken, so there would be renaming needed, but just throwing that thout out there.

Reply:
That makes sense. I didn't notice that I named it differently. Changed both of them to `peer`. Thanks!